modal_proto/api.proto

```diff
@@ -720,6 +743,102 @@ message ContainerExecWaitResponse {
   bool completed = 2;
 }
 
+message ContainerFileCloseRequest {
+  string file_descriptor = 1;
+}
+
+message ContainerFileDeleteBytesRequest {
+  string file_descriptor = 1;
+  optional uint32 start_inclusive = 2;
+  optional uint32 end_exclusive = 3;
+}
+
+message ContainerFileFlushRequest {
+  string file_descriptor = 1;
+}
+
+message ContainerFileFlushResponse {
```

Comment:
Everything here looks good except looks like there are a few of these responses laying around that aren't used anymore. Remove these and I'll LGTM